### DISCUSSION TITLE
Unable to update meeting address. Showing incorrect town when I edit and update meeting it reverts back to incorrect address

### DISCUSSION THREAD

**Discussion by JoeBryan7873** [code4recovery/12-step-meeting-list#1197] [Q&A]
Hi,

I'm running into an issue with a meeting with an incorrect address. The meeting is listed with an incorrect town and when I go into edit the meeting and update with the correct address it reverts back to the incorrect town again. Can't seem to figure out what's causing the issue.

Here is a link to the meeting listing in question. I am currently using the meeting notes to alert people but would like to resolve the issue
https://capecodaa.net/meetings/bass-river-morning-group-3/

Any help would be greatly appreciated,
Thank you
Joe

**Comment by joshreisner** [comment]
hey @JoeBryan7873 the google service we use for address lookups is amazing 99.9% of the time, but there are a few that are just wrong, sorry about that. we have a [little help article about this](https://wordpress.org/plugins/12-step-meeting-list/#i%20need%20to%20correct%20a%20meeting%20address%20or%20change%20a%20pin%27s%20location)

add this code to your theme's functions.php file, then go back and edit the meeting, and paste in the address `288 Main St, West Dennis, MA 02670, USA`

```php
if (function_exists('tsml_custom_addresses')) {
    tsml_custom_addresses(array(
        '288 Main St, West Dennis, MA 02670, USA' => array(
            'formatted_address' => '288 Main St, West Dennis, MA 02670, USA',
            'city' => 'West Dennis',
            'latitude' => 41.6647071,
            'longitude' => -70.1705627,
            'approximate' => 'no',
        ),
    ));
}
```

**Reply by JoeBryan7873** [reply]
Thank you so much I will give that a shot
